lib/tasks/gettext.rake

```diff
@@ -6,9 +6,16 @@ namespace :gettext do
   end
 
   def clean_dir(dir)
-    Dir.glob("#{dir}/*/app.po") do |po_file|
-      GetText::msgmerge(po_file, po_file, 'alaveteli',
-                        :msgmerge => [:sort_output, :no_location, :no_wrap])
```

Comment:
> 1. a bug where new lines where outputted as "\\n" has now been fixed
> …
> This means all our PO files will see updates next time translations are
> imported.

Need to understand what to do about this

Reply:
As you'll see on https://gist.github.com/gbp/dc64bebd09d4a00401b6d33c8a4cff70 strings with newlines appear as being changed.

This as far as I can see is the correct format for PO files now. What we had previously worked and may well continue to work so if we really want to `no-wrap` for the whole PO then we could regexp them out here: https://github.com/mysociety/alaveteli/pull/5779/commits/ed4ca07a861be6c22fc0d00c7a9316c4727bb2d8#diff-faeefb9d492cbafa96677b194d99f752R17

Reply:
Yeah seems worth doing the Right Thing, but just want to understand/check if there are any implications we'll care about, or whether transifex will handle it all transparently.